### DISCUSSION TITLE
Hpool

### DISCUSSION THREAD

**Discussion by dorkmo** [Chia-Network/chia-blockchain#1965] [General]
just thought we could discuss hpool and what, if anything, it means for chia

the pool's share of all new coin awards has steadily increased to about 20% with about 80pb currently
https://www.chiaexplorer.com/blockchain/address/xch1f0ryxk6qn096hefcwrdwpuph2hm24w69jnzezhkfswk0z2jar7aq5zzpfj

https://hpool.com/statistics/chia

if theyre all sharing the same farming address, the participants are just trusting the main operator to fairly distribute awards? perhaps they have a secure system for this built in? 

since they are distributing their own chia software, could this potentially lead to a 51% attack in the future if they continue their growth?

It appears they have about 150 more pb in various poc coins. seems some of this could quickly decide to shift to chia if the price becomes favorable. if about 100pb moved they'd be at 50%.

not trying to fear monger about 51% stuff lol. are there any other things to consider about large pools on the network?

**Comment by dorkmo** [comment]
Ive been keeping an eye on the other coins and it looks like a little over 10PB has come off of hpool's BHD pool size over the last week. Also looks like BHD in general has shrunk around 100PB.

![image](https://user-images.githubusercontent.com/1923070/115901031-4e0b2d80-a426-11eb-9a35-c4f1c76e492f.png)

LitecoinHD has a small but similar move

![image](https://user-images.githubusercontent.com/1923070/115901457-d2f64700-a426-11eb-8686-6c6f8ece0277.png)

Chia still accelerating all around. looks like Hpool will be at around 30% soon.

![image](https://user-images.githubusercontent.com/1923070/115901943-75162f00-a427-11eb-9d52-93b8c5c35f41.png)

**Reply by wallyhall** [reply]
I think regarding hpool, the other factor to consider is the lack of official pooling currently in Chia.  I can fully understand a lot of smaller farmers "giving up" simply because they're feeling like they've seen nothing for their efforts - which will only (by implication) increase the share held by hpool.

Adding official pooling may well help slow hpool's impact.

**Comment by dorkmo** [comment]
thought i might as well do another friday hpool update lol

![image](https://user-images.githubusercontent.com/1923070/116713099-18bc8d80-a99a-11eb-8016-4b2d472f49a1.png)

im not sure why, but the percent of rewards going to the main hpool address seems to always be smaller than the percent of netspace shown on their website.
today, chia explorer shows about 30% of the last 10,000 block awards went to the ...zzpfj address. while doing the math on space gives 516/1255 = 41%

based on space, over the last week they gained about 8% share of total space. but based on rewards, they only gained 1 or 2%.

not sure if theyve changed the way they assign addresses?
I noticed some people on github have asked about copy and pasting plots and double farming them, im wondering if some pool members did that, would it report back as actual space to hpool but not actually win more rewards?

If trends remain the same, i'd predict they'll be showing 50% of space near Monday May 10

**Reply by LeKame** [reply]
I think Chia may fail to crease a decentralized network if this issue is not well considered and solved. Small farmers (those will around 1-2TB hard disks) will give up as they see almost "no chance" to win a reward after days (weeks) of farming.

Hope Chia network will provide some more reasonable way to win a reward for "small" farmers.

**Comment by fabitom** [comment]
hpool need only 130PB to get 51%. So that is 1000% centralization done now. I don't want to say it, but if nobody do anything, I don't see good feature for chia project. If self-pool users will give up bc the low chance of winning, thats end.

**Reply by rpower-jjb** [reply]
I'm confused, but that is normal for me. As of 4/30/21 21:24 UTC network space is showing 1.283 EiB (1312 PiB). 50% is 656 PiB. Also I understand from Reddit that the solo plots made by 1.1.2 will *not* work with the official pooling system.

Question: Will "official" pooling be supported in June, 2021?

Question: Will the current plotter work with that system?

Question: If not, when will a plotter be released that will?

**Comment by dorkmo** [comment]
Very interesting weekend. Looks like 1.1.5 broke hpool's website analytics.

With the recent frozen block peer meltdown, it looked like hpool may have been boosted to 50%. But then we lost lost new data from hpool, so its hard to know how much they were experiencing the same issues as everyone else.

![image](https://user-images.githubusercontent.com/1923070/117865073-20621900-b25b-11eb-9563-03c21847db50.png)

their wallet zzpfj is getting about 37% now, which is the biggest spike we've seen, but it might simply be because of the glitch. will be interesting to see what happens over this week.

interestingly theyve transferred about 75% of their gross winnings out of the main wallet. Not enough data on chia explorer to know where its going, direct distributions or just to a colder wallet?

**Reply by fabitom** [reply]
mainnet space grows faster than hpool and 1-2 week for official pools, so probably users not joining hpool and waiting for official pools (they will get better profits), so everything starts to stabilize imho.

**Reply by secco04** [reply]
i would say to ban hpool from the project since its clearly a scam and the people are dumb as f*ck giving their keys to them

**Comment by dorkmo** [comment]
Interesting statement posted on hpool homepage talking about people double mining on uupool. I think this might explain the discrepancy of percent of reported size vs percent of awards being won.


> Announcement on the follow-up processing of Hpool and UUpool dual mining Chia banned accounts:
> 
> On May 5, 2021, Hpool-Chia pool and UUpool conducted a cross-verification comparison of online submitted data, and banned users who were involved in dual mining in the two mining pools. During this period, many users expressed that they had already expressed their regret for the double mining behavior and were willing to return the extra income to the mining pool, hoping to have the opportunity to enter the mining pool again. For this reason, Hpool decided to unblock dual mining users without harming the interests of miners in the original pool.
> 
> 1. Pre-deduct 50% of the mining revenue of dual-mining users from May 1 to May 5, 2021, given that the 100% of the revenue deducted for blocked users on the evening of May 5 has been injected into the PPLNS pool and distributed to Mining pool users, the extra 50% of the profits reserved for the current dual mining users after deduction will be borne by the early income of the mining pool.
> 2. It is estimated that the deduction will be completed at 24:00 on May 12th.
> 3. If you have any objections to this unblocking plan, you can send an email to Hpool unblocking complaint mailbox: hpool_complain@163.com
> 
> Mail format:
> Title: [Hpool&UUpool Unblocking Appeal], the mining pool account number, reason for appeal, description of the situation, etc. are marked in the text.
> Note: If the title format is incorrect, no reply will be made.
> 
> 
> After that, Hpool and UUpool will conduct a normalized double-mining data comparison. If double-mining is found again, the ID will be permanently blocked.
